docs/TOKENS.md

```diff
@@ -162,14 +182,33 @@ Additional branded values ("secondary", "tertiary", "accent", "supports", etc) c
 
 Defines text highlight values
 
-### Forms
+### Forms <a name="tier-2-forms"></a>
 
 Defines the border width, radius, and color properties for form controls. These values are all mapped to other Tier 2 values and are considered optional in case remapping is required.
 
-### Shadows
+### Shadows <a name="tier-2-shadows"></a>
 
 - `box-shadow` defines the shadow value that gets applied by default to buttons and other "elevated" components like cards.
 
 ## Tier 3 component variables
 
 Tier 3 component variables represent each component's specific values.
+
+## Tailwind Class Tokens
+
+If the EDS tailwind config theme is being used, Tier 2 and tier 3 color tokens are available as a part of tailwind utility classes. Background, border, and text colors will only be available for themselves specifically. e.g.:
+
+```jsx
+<div
+  className="
+    bg-brand-grape-100 // will NOT work since tier 1 colors are not available
+    bg-button-icon-brand // will work since button colors are available across all color utility classes
+    bg-link-neutral // will NOT work since border colors are available for only border color utility classes
+    border-link-neutral // will work since border colors are available for border color utility classes
```

Comment:
`border-link-neutral` is the token name and I'm trying to say that since it's border specific, won't work to apply it as a background color...
Lemme know if the wording doesn't seem intuitive and am super open to suggestions regarding

Reply:
Hmm, I'm still confused since I though `bg-link-neutral` meant "background link neutral" but then the text mentions border and not background, "// will NOT work since border"

Reply:
Lemme know if the new comment seems clearer